site/about/styles.css

```diff
@@ -46,8 +142,27 @@
 }
 
 .school-info-nav {
-    margin-top: 20px;
-    margin-bottom: 30px;
+    margin-left: 20px;
+}
+
+.item-1-3 {
+    padding: 30px;
+    color: gray;
+}
+
+.item-2-3 {
+    padding: 30px;
+    color: gray;
+}
+
+.item-3-3 {
+    padding: 30px;
+    color: gray;
+}
+
+.item-4-3 {
+    padding: 30px;
+    color: gray;
```

Comment:
You can just have 1 class name for all these since they have the same styles. No need for 4 different ones @Merciless0